### PR TITLE
Added a containerized version of docker-compose with Swagger and PostgREST

### PR DESCRIPTION
**Pull request by tsptoni**

### CODE REVIEW

File: environment [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+PGHOST=covid_postgres
+
+PGUSER=covid19_user
+PGPASSWORD=covid_19
+PGDATABASE=covid_19
+PGHOST=covid_postgres
+PGPORT=5432
+
+PGDATA=/var/lib/postgresql/data/pgdata
```
Comment (jqnatividad) [inline environment]: Little nitpick: newline

Comment (jqnatividad) [reply environment]: Otherwise, amazing contribution!  Thanks @tsptoni!

**Review by jqnatividad — APPROVED** [review]
Thanks again for this valuable contribution!

**Comment by ufo56** [comment]
Postgresql docker error
2020-03-17 00:11:35.854 UTC [201] FATAL:  database "covid_19" does not exist

**Pull request merged**